scc/carvel-package-supply-chain.hbs.md

```diff
@@ -213,6 +223,18 @@ Set this parameter by modifying `workload.spec.params.carvel_package_name_suffix
 
 You now see a Carvel `Package` stored in your GitOps repository. For example, at the path `tanzu-java-web-app.default.tap/packages/20230321004057.0.0.yaml` you see a valid Carvel `Package` definition.
 
+###  Git commit SHA in Package version
```

Comment:
Does the user need to do any configuration to use this feature or is this on by default?

Reply:
on by default

Reply:
Consider adding "by default" to the beginning of the sentence so that the user knows this.

"By default, the Git revision identifier for the software included in the Carvel Package is added to the package version, in the format `+build.${git_commit}`, in the following cases:

Reply:
done